drivers/disk/Kconfig.sdmmc

```diff
@@ -47,7 +47,10 @@ config SDMMC_STM32
 config SDMMC_STM32_HWFC
 	bool "STM32 SDMMC Hardware Flow control"
 	depends on SDMMC_STM32
-	depends on SOC_SERIES_STM32H7X || SOC_SERIES_STM32F7X || SOC_SERIES_STM32L4X
+	depends on SOC_SERIES_STM32H7X || \
+		   SOC_SERIES_STM32F7X || \
+		   SOC_SERIES_STM32L4X || \
+		   SOC_SERIES_STM32L5X
```

Comment:
Can you replace by a check on "st,stm32-sdmmc" using dt_compat_enabled? This will avoid adding the same modification each time a new series is added to this driver.

Reply:
ok, just seeing that this is `HWFC` no the driver symbol. Please discard this comment.